netbox_librenms_plugin/templates/netbox_librenms_plugin/inc/_sync_tab_link.html

```
@@ -0,0 +1,13 @@
+<a class="nav-link{% if current_tab == tab_id %} active{% endif %}{% if tab_status.snapshot_available %} sync-cache-ready{% elif tab_status.attention_required %} sync-cache-unavailable{% endif %}" id="{{ tab_id }}-tab"
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)